src/quickstart/provider/azure-acs/boot.md

```diff
@@ -0,0 +1,200 @@
+[
+  {
+    "cloudName": "AzureCloud",
+    "id": "57849302-a9f0-4908-b300-31337a0fb205",
+    "isDefault": true,
+    "name": "Azure Subscription",
+    "state": "Enabled",
+    "tenantId": "591acccc-dddd-4620-8f21-dbbeeeefee21",
+    "user": {
+      "name": "jhansen@deis.com",
+      "type": "user"
+    }
+  }
+]
+```
+
+Replace the value of SUBSCRIPTION_ID with the desired subscription id where you want to deploy from the previous step.  We also set the active subscription to deploy to.
+```
+SUBSCRIPTION_ID=57849302-a9f0-4908-b300-31337a0fb205
+$ az account set --subscription="${SUBSCRIPTION_ID}"
+```
+
+## Create an Azure Service Principle
+
+Next, create an Azure Service Principle that will be used to provision the ACS Kubernetes Cluster. Service Principles are entities that have permission to create resources on your behalf. New Service Principles must be given a unique name, a role, and an Azure subscription that the Service Principle may modify.
+
+```
+$ export SP_JSON=`az ad sp create-for-rbac -n="http://acsk8sdeis" --role="Contributor" --scopes="/subscriptions/${SUBSCRIPTION_ID}"`
+$ export SP_NAME=`echo $SP_JSON | jq -r '.name'`
+$ export SP_PASS=`echo $SP_JSON | jq -r '.password'`
+$ export SP_TENANT=`echo $SP_JSON | jq -r '.tenant'`
+$ echo $SP_JSON
+```
+
+This should display an output similar to this. `jq` has also automatically extracted these values for use in the creation of the cluster.
+```
+{
+  "appId": "58b21231-3dd7-4546-bd37-9df88812331f",
+  "name": "http://workflow-on-acs",
+  "password": "349d4728-438a-52a5-ad25-a740aa0bd240",
+  "tenant": "891a9ddc-477a-4620-8f21-db22ffd3ffea"
+}
+```
+
+## Create Your ACS Kubernetes Cluster
+
+You can build the Kubernetes cluster on ACS using primarily the Azure Web Portal (UI) or entirely using the Azure command line (CLI).  Choose one of the two paths:
+
+### Path 1: Azure 'az' CLI
+
+1. Create an empty Azure resource group to deploy your cluster. The location of the resource group value can be changed to any datacenter.  `az account list-locations` gives the name of all locations.
+
+```
+$ export RG_NAME=myresourcegroup
+$ az resource group create --name "${RG_NAME}" --location southcentralus
+```
+
+2. Execute the command to deploy the cluster. The dns-prefix and ssh-key-value must be replaced with your own values.
+
+```
+$ az acs create --resource-group="${RG_NAME}" --location="southcentralus" \
+  --service-principal="${SP_NAME}" \
+  --client-secret="${SP_PASS}" \
+  --orchestrator-type=kubernetes --master-count=1 --agent-count=2 \
+  --agent-vm-size="Standard_D2_v2" \
+  --admin-username="k8sadmin" \
+  --name="k8sanddeis" --dns-prefix="mydnsprefix" \
+  --ssh-key-value @/home/myusername/.ssh/id_rsa.pub
+```
+
+> Note: When this is successfully executed, you'll only see this to start: `waiting for AAD role to propogate.done`.  It will take a few minutes for the cluster to complete creation.
```

Comment:
We can use markdown admonitions here:

```
!!! note
    When this is successfully executed, you'll only see this to start: `waiting for AAD role to propogate.done`.  It will take a few minutes for the cluster to complete creation.
```